common/txmgr/types/tx_store.go

```diff
@@ -33,8 +33,8 @@ type TxStore[
 	TxHistoryReaper[CHAIN_ID]
 	TransactionStore[ADDR, CHAIN_ID, TX_HASH, BLOCK_HASH, SEQ, FEE]
 
-	// Find confirmed txes beyond the minConfirmations param that require callback but have not yet been signaled
-	FindTxesPendingCallback(ctx context.Context, blockNum int64, chainID CHAIN_ID) (receiptsPlus []ReceiptPlus[R], err error)
+	// Find confirmed txes with finalized state that require callback but have not yet been signaled
```

Comment:
nit: To avoid any confusion between the different state names, we can refer to these as finalized txes
```suggestion
	// Find finalized txes that require callback but have not yet been signaled
```